epub33/explainers/EPUB33-explainer.md

```diff
@@ -0,0 +1,114 @@
+# EPUB 3.3 Explainer
+
+**Written**: 2021-02-22, **Updated**: 2021-02-22
+
+**EPUB 3.3** is a packaging mechanism for web content, designed for electronic books. 
+
+
+
+Spec references:
+* [EPUB 3.3 Core](https://w3c.github.io/epub-specs/epub33/core//), the specification of the file format
+* [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/), the specification for user agents, known as EPUB Reading Systems. 
+* [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/)
+* [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
+
+
+
+
+## Security and Privacy Questionnaire
+
+EPUB files are presented to end users by EPUB Reading Systems, which can be constructed as web applications but are more likely to be native apps. The EPUB specifications define the file format in great detail, but generally give user agents wide latitude, subject mostly to the requirements of the embedded technologies, such as HTML, SVG, and CSS. 
+
+This makes it hard to think about security issues. In order to read EPUB files, I generally have to give personally-identifying information like a credit card number to the ebook retailer. They have likely substantially altered the original EPUB, for example by applying digital rights management to the constituent files. Most of the security implications around EPUB depend on the architectural details and the business model of the reading system. 
+
+It's hard to even reason about how HTML's security issues affect EPUB. If I read an EPUB on the Google Play app on an Android tablet, what risks am I exposed to? Can any of those risks be mitigated by the design of the EPUB specification? 
+
```

Comment:
Again, let us avoid statements that may turn back on us. I would certainly 'soften' the first sentence, but I wonder whether this whole paragraph could be dropped.

We should also make it clearer (and more upbeat) that the security/privacy implication of handling individual content files (HTML, SVG) are based on the security/privacy features of those formats. We should emphasize that the EPUB specification in no ways softens/weakens those requirements, that we give a clear requirement on what the origins are, etc.